datafusion/core/src/datasource/physical_plan/parquet/page_filter.rs

```diff
@@ -276,6 +281,14 @@ fn rows_skipped(selection: &RowSelection) -> usize {
         .fold(0, |acc, x| if x.skip { acc + x.row_count } else { acc })
 }
 
+/// returns the number of rows not skipped in the selection
+/// TODO should this be upstreamed to RowSelection?
```

Comment:
This looks the same as https://docs.rs/parquet/latest/parquet/arrow/arrow_reader/struct.RowSelection.html#method.row_count

It would be great to upstream this and rows_skipped to `parquet` -- any chance you are willing to file a ticket to do so?

Reply:
sent to https://github.com/apache/arrow-rs/pull/6429